### PR TITLE
Relax label constraints

### PR DESCRIPTION
**Pull request by noamzbr**
Remove the requirement for consecutive labels

### CODE REVIEW

File: deepchecks/checks/distribution/trust_score_comparison.py [hunk 1]
```diff
@@ -130,15 +130,23 @@ def run(self, train_dataset, test_dataset, model=None) -> CheckResult:
             max_num_categories=self.max_number_categories
         )
```
Comment (nirhutnik) [inline deepchecks/checks/distribution/trust_score_comparison.py:132]:
```suggestion
# Trust Score model expects labels to be consecutive integers from 0 to n-1, so we transform our label to this format:
```

**Review by nirhutnik — APPROVED** [review]

**Review by nirhutnik — APPROVED** [review]
